### DISCUSSION TITLE
Filtering Data Provider using OR condition

### DISCUSSION THREAD

**Discussion by mathewhealthi** [Budibase/budibase#5920] [Help]
I'm trying to filter my data provider so that it displays rows from my table that has either of the two statuses as shown in the image. However, Budibase seems to use an AND functionality and hence when I put more than one filter criteria, it doesn't show me any results since no row satisfies both conditions.

Is there a way for me to filter based on an OR condition?

![image](https://user-images.githubusercontent.com/105265941/168742639-04bf9c92-b687-47f7-bebb-1b3358e4addb.png)

**Comment by melohagan** [comment]
Hi @mathewhealthi 

Unfortunately not, there is a feature request open: https://github.com/Budibase/budibase/issues/3863
